src/ripple/rpc/handlers/Subscribe.cpp

```diff
@@ -201,6 +201,24 @@ doSubscribe(RPC::JsonContext& context)
         JLOG(context.j.debug()) << "doSubscribe: accounts: " << ids.size();
     }
 
+    if (context.params.isMember(jss::account_history_tx_stream))
+    {
```

Comment:
I think we should charge consumers for this call. We can do that by just adding:
```
context.loadType = Resource::feeMediumBurdenRPC;
```

Reply:
Thanks, good idea, added.